aviary/interface/default_phase_info/two_dof_fiti_copy.py

```diff
@@ -0,0 +1,220 @@
+from aviary.utils.aviary_values import AviaryValues
```

Comment:
It is another instance of something that only exists because of the difference in the way the trajectory is specified depending on whether or not submodel comps can be used.